lookout/style/typos/tests/test_preprocessing.py

```diff
@@ -0,0 +1,28 @@
+import tempfile
+import unittest
+
+import pandas
+
+from lookout.style.typos.preprocessing import print_frequencies
+
+
+class PreprocessingTest(unittest.TestCase):
+    def test_print_frequencies(self):
+        frequency_column = "num_occ"
+        id_stats = pandas.DataFrame([[23], [6], [17]], columns=[frequency_column],
+                                    index=["get", "token", "set"])
+        vocabulary = {"get", "set"}
+        vocabulary_file = tempfile.NamedTemporaryFile()
```

Comment:
always use 
```python
with tempfile.NamedTemporaryFile() as vocabulary_file:
```
and no `vocabulary_file.close()` in the end. 
It helps you to avoid bad situations with exceptions+unclosed files.